preview/index.html

```diff
@@ -4,13 +4,11 @@
     <meta charset="utf-8">
     <title>Boxbits</title>
 
-    <link rel="stylesheet" href="reset.css" />
     <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" type="text/css">
   </head>
   <body>
     <div id="container"></div>
 
-    <script type="text/javascript" src="bundle.js"></script>
-    <script type="text/javascript" src="example.js"></script>
```

Comment:
The two script tags were intentional: one for the lib, another for the separate example as "client project code"